src/panoptes/pocs/camera/camera.py

```diff
@@ -559,19 +562,21 @@ def process_exposure(self,
         Raises:
             FileNotFoundError: If the FITS file isn't at the specified location.
         """
+        # Wait for exposure to complete. Timeout handled by exposure thread.
+        while self.is_exposing:
+            time.sleep(1)
+
+        self.logger.debug(f'Starting exposure processing for {observation_event}')
+
         # If passed an Event that signals the end of the exposure wait for it to be set
         compress_fits = compress_fits or self.get_config('observations.compress_fits')
```

Comment:
I'm not sure this way (using `or`) of doing things is such a good idea - I would expect keyword arguments to override config values. This may lead to some confusion.

Reply:
That's what this does.  The value from the first argument (`compress_fits`) will be used unless it's value is `None`, in which case the `or` will use the second value (the config lookup).

I've started using this `param=None` then `param = param or something_else` syntax quite a bit and think it's actually a lot cleaner but am open to discussion if others find it confusing.

Reply:
Basically, `None or 42` resolves to `42`.

Reply:
@wtgee what if (for example) `compress_fits=False` is passed to the function, but the config has `observations.compress_fits=True`. In this case, the keyword argument is ignored in favour of the config value?

Reply:
Ah, good one. I'm not sure why I have missed that before. Now I need to check some other code...

Thanks!